.github/workflows/ci-test.yml

```diff
@@ -15,6 +15,10 @@ jobs:
       with:
         submodules: true
 
+    - uses: actions/setup-node@v4
+      with: { node-version-file: .nvmrc }
+    - run: npm install
```

Comment:
With the consolidated packages now at the top level, we need to run `npm install` before we build.